### PR TITLE
add description new creatures in get started pages

### PR DESCRIPTION
**Pull request by Lisofffa**

### CODE REVIEW

File: docs/manifest.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  - `FEATURE` – a dapplet, its main part that interacts with the adapter and Core
+  - `ADAPTER` – a site-specific adapter that allows dapplets to work with site specific contexts
+  - `INTERFACE` – a virtual adapter which provides the interface for dapplets for using several site-specific adapters
+  - `LIBRARY` – a dynamic adapter. It provides the work of the all other adapters. It's specified in the extension's settings
+
+    ![set dynamic adapter](/img/manif_03.jpg)
+
+- **title** – a module's name. It's displayed in the extension's dapplets' list, in the Dapplets Store, on the NFT etc.
+- **icon** – a link to the dapplet's icon. It's an optional parameter.
+- **contextIds** – a list of resources where the module is loaded and activated. You should list site domains or a name of the module (adapter, interface), where domains are already listed.
```
Comment (alsakhaev) [inline docs/manifest.md:61]:
```json
[
  // domains
  "google.com",
  "www.google.com",
  "gateway.ethswarm.org",
  "dapplets.org",
  "127.0.0.1",
  "forum.dapplets.org",
  "youtube.com",
  "www.youtube.com",
  "m.youtube.com",
  "twitter.com",
  "www.twitter.com",
  "mobile.twitter.com",
  "instagram.com",
  "www.instagram.com",
  "github.com",
  "www.github.com",

  // name of adapter
  "twitter-adapter.dapplet-base.eth",
  "common-adapter.dapplet-base.eth",
  "video-adapter.dapplet-base.eth",
  "search-adapter",
  "youtube-adapter",
  "swarm-gateway-adapter",
  "instagram-adapter.dapplet-base.eth",
  "stickery-adapter.dapplet-base.eth",
  "github-adapter-example.dapplet-base.eth",

  // interface name
  "identity-adapter.dapplet-base.eth",

  // adapters, providing stable id
  "youtube.googleapis.com/id",
  "twitter.com/id",
  "www.twitter.com/id",
  "mobile.twitter.com/id",
  "youtube.com/id",
  "www.youtube.com/id",

  // content detectors
  "video",

  // context id (eg. tweet id from adapter)
  "twitter.com/1493947374460452866"
]
```

Comment (alsakhaev) [reply docs/manifest.md:61]: Oh, sorry. I didn't see the text below

File: docs/manifest.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  }
+  ```
+
+  It means that the module works if there are <video\> elements on the page. It is also dynamic context.
+
+- **config** – a dapplet's config. It's an optional field that's used only in dapplets.
+The idea is to add some settings to the dapplet which can be changed in the extension.
+For more information look [here](./#).
+
+- **overlays** – a list of the overlays using with the dapplet. If your dapplet uses the overlay you have to add here its name/ID and the development server:
```
Comment (alsakhaev) [inline docs/manifest.md:112]: `assets-manifest.json` should be available at the root of URL
Example:
```json
{"index.html": "index.html", "styles.css": "css-62d9da.css"}
```

File: docs/manifest.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+- **dependencies** – adapters which are used in the module. You have to set dependencies for FEATURE and ADAPTER modules.
+In the dapplets you set the site-specific and virtual adapters, in the site-specific adapters – dynamic adapter:
+
+  ```json
+  {
+    "dependencies": {
+      "dynamic-adapter.dapplet-base.eth": "0.6.22"
+    },
+  }
+  ```
```
Comment (alsakhaev) [inline docs/manifest.md:133]: Reminder: the following declaration is also allowed theoretically, but it's not tested.
```json
  {
    "dependencies": {
      "twitter-adapter.dapplet-base.eth": {
         "default": "0.6.22",
         "new": "0.6.22"
      }
    },
  }
  ```

**Review by alsakhaev — CHANGES_REQUESTED** [review]

File: docs/manifest.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  }
+}
+```
+
+### Manifest Structure
+
+Some fields refer to the `package.json`. They have `"$ref"` children with the field link values. All these fields are obligatory. Set them in the `package.json`:
+
+- **name** – a name of the module. The name is the ID of your module so it must be unique
+- **version** – a version of the module
```
[the remark this answers is not in the document]
Comment (Ni-2) [reply docs/manifest.md:36]: Add refer. Will add prereleases when it's tested

File: docs/manifest.md (hunk 2 above)
Comment (Ni-2) [reply docs/manifest.md:112]: У меня везде такой манифест получается, поэтому указал его:

```json
{
  "index.html": "index.html",
  "main.js": "main.js"
}
```

File: docs/manifest.md (hunk 3 above)
Comment (Ni-2) [reply docs/manifest.md:133]: Don't think that we need to show it in the docs. Especially it's not tested.

**Pull request merged**